docs/usage.rst

```diff
@@ -158,5 +158,19 @@ Running this will check that your project adheres to the contracts you've define
 
     lint-imports --verbose
 
+Running using pre-commit
+------------------------
+
+``import-linter`` can run as a `pre-commit`_ hook by adding this section to your `.pre-commit-config.yaml` file:
+
+.. code-block:: yaml
+
+  repos:
+  - repo: https://github.com/seddonym/import-linter
+    rev: v2.0
```

Comment:
Am I correct in thinking this should be the latest version of Import Linter?

It would be good if this could be pulled in dynamically somehow from the `release` variable defined in `docs/conf.py`. I tried to figure out how to do this for about ten minutes, not sure how easy this would be. How's your Sphinx?